### ISSUE TITLE
Blueprintable trees and rocks

### ISSUE THREAD

**Issue by Brok010** [jpanther/Dectorio#204]
Idk if I messed smt up or if its just not a thing, but blueprintable trees and rocks would be great.

**Comment by The-Grim-Sleeper** [comment]
I'd love this, but also with cliffs.

**Comment by jpanther** [comment]
I don't believe blueprinting the trees and rocks is possible as they are not considered to be player objects.
